opentelemetry-exporter-gcp-trace/opentelemetry-exporter-gcp-trace.gemspec

```diff
@@ -14,39 +14,37 @@
 
 # frozen_string_literal: true
 
-require_relative "lib/opentelemetry/exporter/gcp/trace/version"
+require_relative "lib/opentelemetry/exporter/google_cloud_trace/version"
 
 Gem::Specification.new do |spec|
   spec.name = "opentelemetry-exporter-gcp-trace"
-  spec.version = Opentelemetry::Exporter::Gcp::Trace::VERSION
+  spec.version = Opentelemetry::Exporter::GoogleCloudTrace::VERSION
   spec.authors = ["Nivedha"]
   spec.email = ["nivedhasenthil@gmail.com"]
 
-  spec.summary = "TODO: Write a short summary, because RubyGems requires one."
-  spec.description = "TODO: Write a longer description or delete this line."
-  spec.homepage = "TODO: Put your gem's website or public repo URL here."
+  spec.summary = "Opentelemetry exporter for Google Cloud Trace"
+  spec.description = "opentelemetry-exporter-gcp-trace is the officially supported exporter for Google Cloud Trace"
+  spec.homepage = "https://github.com/GoogleCloudPlatform/opentelemetry-operations-ruby/tree/main/opentelemetry-exporter-gcp-trace"
   spec.required_ruby_version = ">= 2.6.0"
-
-  spec.metadata["allowed_push_host"] = "TODO: Set to your gem server 'https://example.com'"
-
   spec.metadata["homepage_uri"] = spec.homepage
-  spec.metadata["source_code_uri"] = "TODO: Put your gem's public repo URL here."
-  spec.metadata["changelog_uri"] = "TODO: Put your gem's CHANGELOG.md URL here."
+  spec.require_paths = ["lib"]
+  spec.license = "Apache-2.0"
 
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
-  spec.files = Dir.chdir(__dir__) do
+  spec.files = Dir.chdir __dir__ do
     `git ls-files -z`.split("\x0").reject do |f|
       (f == __FILE__) || f.match(%r{\A(?:(?:bin|test|spec|features)/|\.(?:git|circleci)|appveyor)})
     end
   end
-  spec.bindir = "exe"
-  spec.executables = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
-  spec.require_paths = ["lib"]
 
-  # Uncomment to register a new dependency of your gem
-  # spec.add_dependency "example-gem", "~> 1.0"
+  spec.add_dependency "google-cloud-core", "~> 1.5"
+  spec.add_dependency "google-cloud-trace-v2", "~> 0.0"
+  spec.add_dependency "opentelemetry-sdk"
```

Comment:
Use `opentelemetry-api` as the dependency. (Although you can use `opentelemetry-sdk` in your Gemfile for use by tests.)

Reply:
As discussed offline, since [SDK](https://opentelemetry.io/docs/reference/specification/overview/#sdk) interface is to be used for Plugin interface we will stick with having it as dependency for now.